src/containers/home/home.component.jsx

```diff
@@ -1,49 +1,54 @@
-import React from "react";
-
+import React, { useEffect, useState } from "react";
 import { auth } from "database";
-
-import { fetchPosts } from "database/posts";
-import { addPost } from "database/addPost";
-
 import { useSelector } from "react-redux";
+import CreatePost from "containers/create-post/create-post.component";
```

Comment:
```suggestion
import CreatePost from "containers/create-post";
```

The `index.js` should export the component file and enable us to import this way

Reply:
Theres no Index.js for create-post so it cannot be imported like this

Reply:
Lol you can make one! Just follow the same pattern as in the `Home` folder